sys/vm/vm_page.h

```diff
@@ -434,6 +445,13 @@ extern struct mtx_padalign pa_lock[];
  * PGA_SWAP_FREE is used to defer freeing swap space to the pageout daemon
  * when the context that dirties the page does not have the object write lock
  * held.
+ *
+ * PGA_CAPDIRTY indicates that a capability was written to this page since the
+ * last time this bit (and the underlying architectural permission) was
+ * cleared.  Setting this bit happens without the page lock held, but clearing
+ * it requires synchronization.  Specifically, this bit may be cleared only if
+ * the page is xbusy and the clearer subsequently measures all tag bits on the
+ * page.
```

Comment:
Part of this comment makes it seem like CAPDIRTY corresponds to an actual dirty bit (tracks cap stores since the last time it was cleared), but the part about requiring measurement of all tag bits on the page makes it seem like it corresponds to MCC (the page is permitted to contain capabilities and in fact is known to maybe contain capabilities). The code seems to be using CAPDIRTY like an actual dirty bit - e.g. init_pte_prot and pmap_is_capdirty. I think it would be useful to introduce another flag that corresponds to MCC.